packages/liveblocks-yjs/package.json

```diff
@@ -24,7 +24,7 @@
   ],
   "scripts": {
     "dev": "tsup --watch",
-    "build": "tsup && cp dist/index.d.ts dist/index.d.mts",
+    "build": "tsup && shx cp dist/index.d.ts dist/index.d.mts",
```

Comment:
Hmm. This `cp` should no longer be necessary with the latest version of `tsup`. I'll have a look at the configuration and upgrade this.

Reply:
Fixed!